rclpy/rclpy/__init__.py

```diff
@@ -192,6 +192,8 @@ def spin_once(node: 'Node', *, executor: 'Executor' = None, timeout_sec: float =
     It is possible the work done is for a node other than the one provided if the global executor
     has a partially completed coroutine.
 
+    This method should not be called from multiple threads.
+
```

Comment:
How about

```
This method should not be called from multiple threads with the same node argument or executor argument.
```

It leaves out the details of why (avoiding calling `rcl_wait` on the same entity from multiple threads), but I think that's fine for such a high level method.

Reply:
Sure. If you make this a suggestion, I'd be happy to accept it. (I don't have a local copy of this very old PR any longer)

Reply:
I added it as a new commit!